ORStools/proc/isochrones_layer_proc.py

```diff
@@ -278,18 +189,24 @@ def postProcessAlgorithm(self, context, feedback):
 
         return {self.OUT: self.dest_id}
 
-    def get_sorted_feature_parameters(self, layer):
+    @staticmethod
+    def get_sorted_feature_parameters(layer, id_field_name: str):
         """
         Generator to yield geometry and id of features sorted by feature ID. Careful: feat.id() is not necessarily
         permanent
 
         :param layer: source input layer.
+        :param id_field_name: layer field containing id values
```

Comment:
This should be documented independently from `layer` and have a type as well.

Reply:
type is in the definition (id_field_name: str). How independently?

Reply:
When documenting params and their types in docstrings, there is usually a line of whitespace between parameter-type-pairs.
Here, the ordering of 
```
:param layer:
:param id_field_name:
:type layer:
```
is a bit weird.

For consistency, either the layer type should be moved to the definiton as well or this should be ordered differently.

Reply:
done